scripts/build/build/targets.py

```diff
@@ -77,6 +77,8 @@ def BuildHostFakeTarget():
         "-clang").ExceptIfRe('-ossfuzz')
     target.AppendModifier("ossfuzz", fuzzing_type=HostFuzzingType.OSS_FUZZ).OnlyIfRe(
         "-clang").ExceptIfRe('-libfuzzer')
+    target.AppendModifier("pw-fuzztest", fuzzing_type=HostFuzzingType.PW_FUZZTEST).OnlyIfRe(
+        "-clang").ExceptIfRe('-(libfuzzer|ossfuzz)')
```

Comment:
should we add a `ExceptIfRe` on asan? (i.e. both here and deny asan if pw-fuzztest is set)? 

Your docs seem to say that asan is auto-enabled for this.